filebeat/inputsource/tcp/client.go

```diff
@@ -18,7 +19,7 @@ type client struct {
 	done           chan struct{}
 	metadata       Metadata
 	splitFunc      bufio.SplitFunc
-	maxReadMessage size
+	maxReadMessage common.Size
```

Comment:
Could we rename the variable here to be in line with the config naming? This will make it easier to read the code base.

Reply:
Will rename it to `maxMessageSize`